src/client/js/components/circle-chart-test-data.js

```diff
@@ -0,0 +1,250 @@
+export default [
+  {
+    title: 'Chart title 1',
+    data: [
+      {
+        key: 'resolved',
+        name: 'Resolved',
+        count: 0,
+        color: '#9059ff'
+      },
+      {
+        key: 'unresolved',
+        name: 'Unresolved',
+        count: 10,
+        color: '#321c64'
+      }
+    ],
+    showPercentageFor: 'resolved',
+    isDonut: true
+  },
```

Comment:
Presumably we will be deleting this file and the associated test changes on the landing page?  Assuming yes, it would still be nice to keep a schema example in comment form within the component.  A usage example might be nice too, specifically demonstrating the `JSON.stringify` aspect of the data.